### PR TITLE
Add some files for grpc auth tutorial

### PR DESCRIPTION
**Pull request by qiwzhang**
api_config_auth.yaml to specify authentication
google_jwt_client.py: to generate JWT token from a service account file.

### CODE REVIEW

**Comment by qiwzhang** [comment]
@jonparrott or @waprin 
could you help to review it?

File: endpoints/bookstore-grpc/google-jwt-client.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
```
Comment (jonparrott) [inline endpoints/bookstore-grpc/google-jwt-client.py]: Why not go ahead and use `google-auth`:

```python
import google.auth.crypt
import google.auth.jwt

with open(service_account_filename, 'r') as fh:
    service_account_info = json.load(fh)

signer = google.auth.crypt.Signer.from_string(
    service_account_info['private_key'], service_account_info['private_key_id'])

payload = {
    ...
}

jwt = google.auth.jwt.encode(signer, payload)
```

Comment (qiwzhang) [inline endpoints/bookstore-grpc/google-jwt-client.py]: Done

File: endpoints/bookstore-grpc/jwt_token_gen.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import argparse
+import json
+import time
+
+import google.auth.crypt
+import google.auth.jwt
+
+"""Max lifetime of the token (one hour, in seconds)."""
+MAX_TOKEN_LIFETIME_SECS = 3600
+
```
Comment (jonparrott) [inline endpoints/bookstore-grpc/jwt_token_gen.py]: two new lines between constants and function definitions.

Comment (qiwzhang) [inline endpoints/bookstore-grpc/jwt_token_gen.py]: Done

**Comment by jonparrott** [comment]
Thanks, @qiwzhang!

**Pull request merged**